extensions/test/integration/operation/shoot_operation.go

```diff
@@ -70,7 +70,7 @@ func ScaleDeployment(setupContextTimeout time.Duration, client client.Client, de
 	ctxSetup, cancelCtxSetup := context.WithTimeout(context.Background(), setupContextTimeout)
```

Comment:
Why is this `func` here? Can't we reuse `framework.ScaleDeployment()`?

Reply:
Maybe we can also directly move the other function into `test/framework`...

Reply:
the function is already in the framework so i just removed the function in the extension test and use the framework function.